tests/playwright/src/specs/kind.spec.ts

```diff
@@ -121,21 +128,43 @@ describe('Kind End-to-End Tests', async () => {
       );
       test('Check resources added with the Kind cluster', async () => {
         const containersPage = await navigationBar.openContainers();
-        await playExpect.poll(async () => containersPage.containerExists(CONTAINER_NAME)).toBeTruthy();
-        const containerDetailsPage = await containersPage.openContainersDetails(CONTAINER_NAME);
+        await playExpect.poll(async () => containersPage.containerExists(KIND_CONTAINER_NAME)).toBeTruthy();
+        const containerDetailsPage = await containersPage.openContainersDetails(KIND_CONTAINER_NAME);
         await playExpect.poll(async () => await containerDetailsPage.getState()).toEqual(ContainerState.Running);
 
         const volumesPage = new VolumesPage(page);
-        const volumeName = await getVolumeNameForContainer(page, CONTAINER_NAME);
+        const volumeName = await getVolumeNameForContainer(page, KIND_CONTAINER_NAME);
         if (!volumeName) {
-          throw new Error(`Volume name for container "${CONTAINER_NAME}" is not defined.`);
+          throw new Error(`Volume name for container "${KIND_CONTAINER_NAME}" is not defined.`);
         }
         const volumeDetailsPage = await volumesPage.openVolumeDetails(volumeName);
         await playExpect.poll(async () => await volumeDetailsPage.isUsed()).toBeTruthy();
       });
       test('Validate correct Kubernetes context is selected', async () => {
         await statusBar.validateKubernetesContext(KUBERNETES_CONTEXT);
       });
+      describe('Deploy a container to the Kind cluster', async () => {
+        test('Pull an image and start a container', async () => {
+          const imagesPage = await navigationBar.openImages();
+          const pullImagePage = await imagesPage.openPullImage();
+          await pullImagePage.pullImage(IMAGE_TO_PULL);
+          await imagesPage.waitForImageExists(IMAGE_NAME, 10000);
+          await imagesPage.startContainerWithImage(IMAGE_NAME, CONTAINER_NAME);
+        });
+        test('Deploy the container ', async () => {
+          const containerDetailsPage = new ContainerDetailsPage(page, CONTAINER_NAME);
+          await playExpect(containerDetailsPage.heading).toBeVisible();
+          await playExpect(containerDetailsPage.deployButton).toBeVisible();
```

Comment:
AS far as I know the method startContainersWithImage method returns a ContainersPage object, which indicates that is the page that should be currently navigated on, yet I see you are creating a container details page here and checking for location, is this correct? What ensure the navigation from containers page to container details page?

Reply:
You are right; the `startContainerWithImage` method returns a `ContainersPage` object. However, when we start a container with an image manually, we navigate to the container details page. Therefore, I think the `startContainerWithImage` method needs to be refactored to return a `ContainerDetailsPage` instead.

Reply:
WDYT @cbr7 ?

Reply:
@amisskii are you certain this is correct? Because there are many tests that use this method and I would expect to be broken if that were true. But if the flow did indeed change than certainly the method should be refactored to return the correct next page yes.

Reply:
At least that is how it works for me :) I will open a new pr.

Reply:
@amisskii I think there might be a difference on if you start the container with interactive terminal or not, if you have terminal it would lead to the details page but if there is no terminal it will navigate the the containers page.